src/main/java/brobot/exception/NoSuchTaskException.java

```diff
@@ -0,0 +1,13 @@
+package brobot.exception;
+
+/**
+ * Represents an exception caused when no such task is found for a command that needs a task number
+ */
+public class NoSuchTaskException extends BroException {
+    /**
+     * Constructor for the exception.
+     */
+    public NoSuchTaskException() {
+        super("Ey bro, no such task lah!");
```

Comment:
Your exceptions are really clear and it is split up properly. And the added humour is greatly appreciated!